.github/workflows/merge-group.yml

```diff
@@ -15,19 +15,30 @@ jobs:
       mergedSha: ${{ github.event.merge_group.head_sha }}
       targetSha: ${{ github.event.merge_group.base_sha }}
 
-  # This job's only purpose is to serve as a target for the "Required Status Checks" branch ruleset.
+  # This job's only purpose is to create the target for the "Required Status Checks" branch ruleset.
   # It "needs" all the jobs that should block the Merge Queue.
-  # If they pass, it is skipped — which counts as "success" for purposes of the branch ruleset.
-  # However, if any of them fail, this job will also fail — thus blocking the branch ruleset.
   no-pr-failures:
+    if: github.event_name != 'pull_request'
```

Comment:
Q: github state the default `if` is `success()`

Does this imply that we should do `github.event_name != 'pull_request' && success()`?

Reply:
> EDIT: I'm not sure why I assumed this.

Ah, I'm mis-remembering this: https://docs.github.com/en/actions/reference/workflows-and-actions/expressions#status-check-functions

> A default status check of `success()` is applied unless you include one of these functions.

So if the expression doesn't use any status-check functions, it'll default to `success() &&`